shinylive/_main.py

```diff
@@ -469,6 +471,154 @@ def defunct_help(cmd: str) -> str:
 """
 
 
+# #############################################################################
+# ## shinylive.io url
+# #############################################################################
+
+
+@main.group(
+    short_help="Create or decode a shinylive.io URL.",
+    help="Create or decode a shinylive.io URL.",
+    no_args_is_help=True,
+    cls=OrderedGroup,
+)
+def url() -> None:
+    pass
+
+
+@url.command(
+    short_help="Create a shinylive.io URL from local files.",
+    help="""
+Create a shinylive.io URL for a Shiny app from local files.
+
+APP is the path to the primary Shiny app file.
+
+FILES are additional supporting files for the app.
```

Comment:
If there are lots of supporting files, this could be tedious to specify from command line? Seems like we should take an `APPDIR` similar to `shinylive export` and `rsconnect::deploytApp()`?

Reply:
Good idea, I've set it up now so that if a file in FILES is a directory, all of the files in that directory will be included in the bundle (recursively).